packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -148,18 +163,28 @@ class AnalyzeCommand extends FlutterCommand {
 
     if (argResults['current-package']) {
       // ./lib/main.dart
-      String mainPath = 'lib/main.dart';
+      String mainPath = path.join('lib', 'main.dart');
```

Comment:
(If it's needed, how do you do constant paths? We have some constructors whose default arguments are paths, but those can't be constant if you have to pass them to `join()`.)